crates/sui-json-rpc/src/api/read.rs

```diff
@@ -204,4 +204,16 @@ pub trait ReadApi {
         /// Checkpoint identifier, can use either checkpoint digest, or checkpoint sequence number as input.
         id: CheckpointId,
     ) -> RpcResult<Checkpoint>;
+
+    /// Return pagenated list of checkpoints
+    #[method(name = "getCheckpoints")]
+    async fn get_checkpoints(
+        &self,
+        /// Optional paging cursor
+        cursor: Option<usize>,
+        /// Maximum item returned per page, default to QUERY_MAX_RESULT_LIMIT if not specified.
+        limit: Option<usize>,
+        /// query result ordering, default to false (ascending order), oldest record first.
+        descending_order: bool,
```

Comment:
@gegaowp has recently changed the definition of cursor to be exclusive in https://github.com/MystenLabs/sui/pull/9017/files ? Can you conform to that definition so that it's consistent with other APIs?


```
/// `next_cursor` points to the last item in the page;
/// Reading with `next_cursor` will start from the next item after `next_cursor` if
/// `next_cursor` is `Some`, otherwise it will start from the first item.
```